modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughConstants.java

```diff
@@ -25,7 +25,9 @@
     public static final String CONNECTION_POOL = "CONNECTION_POOL";
     public static final String TUNNEL_HANDLER = "TUNNEL_HANDLER";
     public static final String CONNECTION_INIT_TIME = "CONNECTION_INIT_TIME";
-    public static final String CONNECTION_RELEASE_TIME = "CONNECTION_RELEASE_TIME";
+    /** time at which the connection is timed out, calculated using the keep alive timeout or connection idle time */
+    public static final String CONNECTION_EXPIRY_TIME = "CONNECTION_EXPIRY_TIME";
+    public static final String CONNECTION_KEEP_ALIVE_TIME_OUT = "CONNECTION_KEEP_ALIVE_TIME_OUT";
```

Comment:
Shall we add a description for this as well

Reply:
Fixed